### PR TITLE
Revert to using dev secrets

### PR DESCRIPTION
**Pull request by adrian-fitsios**
[AB#7006](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7006)

### CODE REVIEW

File: Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml [hunk 1]
```diff
@@ -54,39 +58,54 @@ spec:
...
         - name: REDIS_TLS
-          value: srt-redistls@azurekeyvault
+          value: "true"
 
           ## The password to connect to redis
         - name: REDIS_PASS
-          value: srt-redispass@azurekeyvault
-
-          ## The secret needed for encoding and decoding the cookie
-        - name: COOKIE_SECRET
```
Comment (talawson05) [inline Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml:84]: does order form still need the cookie secret?

Comment (adrian-fitsios) [reply Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml:84]: I would assume so as OF needs to log users in, why?

Comment (talawson05) [reply Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml:84]: I can see that it's not been removed, it's just moved up the file. Thanks for pointing that out @assoomro

**Review by talawson05 — APPROVED** [review]
It's a shame you're reverting the KV change, it was probably the right thing to do if it worked.

**Comment by adrian-fitsios** [comment]
> It's a shame you're reverting the KV change, it was probably the right thing to do if it worked.

Agreed, it would'have been a nice thing to have, however, the flaky injectors are more trouble than its worth.

**Review by stuart-bass-bjss — APPROVED** [review]

**Pull request merged**